src/tests/integration/test-data/show/Enum.enum_storage_range().cse.expected

```diff
@@ -266,6 +266,7 @@ module SUMMARY-TEST%ENUM.ENUM-STORAGE-RANGE():0
        andBool ( ( notBool #range ( 0 < ORIGIN_ID:Int <= 9 ) )
        andBool ( ( notBool #range ( 0 < C_ENUM_MEMBER_CONTRACT_ID:Int <= 9 ) )
                ))))))))))))))))))))))))))))))))))
+       ensures #lookup ( C_ENUM_MEMBER_CONTRACT_STORAGE:Map , 1 ) <=Int 5
```

Comment:
Both constraints (this one and the one in the `requires`) should either be this or `#lookup ( C_ENUM_MEMBER_CONTRACT_STORAGE:Map , 1 ) <Int 6`. What does the compiler check and what do we put in enum constraints? @nwatson22 

This is popping up because the simplifications work slightly differently now that Z3 is used more in the booster.

Reply:
I'm not sure what you mean by what does the compiler check. I'm not 100% sure but I haven't seen it inserting range checks for enum values. I thought this was enforced at compile time. The enum constraints are just added to the initial node for each enum field in each CSE-included symbolic contact account, and they are of the form
```
                    mlEqualsTrue(
                        ltInt(
                            KEVM.lookup(storage_map, intToken(field.slot)),
                            intToken(enum_max),
                        )
                    )
```

Reply:
I think that this is ultimately a minor inconsistency with what we add and what Kore does, which is filter branches post-addition of branching constraints. Once the booster has the syntactic simplifications operational this `ensures` will go away. For now, it is ok for it to stay.